sps-team-38/src/main/webapp/index.html

```diff
@@ -49,7 +49,7 @@ <h3>Request an Volunteer Here!</h3>
                 <option>Spanish</option>
                 <option>Arabic</option>
             </datalist> <br>
-            <label for="meet_link">Paste a link to a Google Meet chat for your interview:</label>
+            <label for="meet_link">Paste a link to a Google Meet session for your volunteer(s) to join:</label>
```

Comment:
No big deal just kind of confused why the change wasn't already there after my PR was merged

Reply:
in resolving the conflict on my branch it probably got lost.